src/components/utils/hooks/useMessagePreviews.tsx

```diff
@@ -65,13 +69,6 @@ const useMessagePreviews = () => {
     }
   });
 
-  const reset = () => {
```

Comment:
What's the reason to move this out to the store? For example, I think we no longer reset the loading state now 👀

Reply:
I saw this was not needed and was not working correctly, so I moved the reset as a store action. I think we don't really need to reset the loading state, as it gets set as it is on page navigation